### PR TITLE
Parameterise import

### PR DESCRIPTION
**Pull request by ellesmith88**
## Overview

This fixes the import error with parameterise and uses the pypi package for roocs-utils

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -24,3 +24,5 @@ dependencies:
 - networkx
 # tests
 - pytest
+- pip:
+    - roocs-utils>=0.1.4
```
Comment (cehbrecht) [inline environment.yml:28]: I would only put conda packages here and keep the packages from pypi only in `requirements.txt`.

The reason for this is to create a conda spec file which can only have conda packages.

Comment (ellesmith88) [reply environment.yml:28]: Ok that makes sense, I'll take it out.

I'm waiting for the clisops import fix before I convert this back to an open PR

**Comment by cehbrecht** [comment]
@ellesmith88 is this one ready to merge?

**Comment by ellesmith88** [comment]
@cehbrecht yes if you're happy with it

**Review by cehbrecht — APPROVED** [review]

**Comment by cehbrecht** [comment]
@ellesmith88 looks fine :) merged.

**Pull request merged**